ai4pkm_cli/commands/process_notes.py

```diff
@@ -0,0 +1,531 @@
+"""Apple Notes processing system with AppleScript integration."""
+
+import subprocess
+import os
+import json
+import glob
+import re
+import shutil
+from rich.console import Console
+
+
+class ProcessNotes:
+    """Handles Apple Notes processing with configurable destination folders."""
+
+    def __init__(self, logger, config=None):
+        """Initialize notes processor."""
+        self.logger = logger
+        self.console = Console()
+        self.config = config
+
+    def process_notes(self, use_cache=False):
+        """Process Apple Notes with configurable destination folder.
+        
+        Args:
+            use_cache (bool): If True, use cached AppleScript export instead of re-running
+        """
+        
+        # Get configurable paths and settings
+        if self.config:
+            destination_folder = self.config.get_notes_destination_folder()
+            days = self.config.get_notes_days()
+        else:
+            # Use defaults if no config object provided
+            destination_folder = "Ingest/Notes/"
```

Comment:
This default values are already defined in `self.config.get_notes_destination_folder()` function, so this code would never hit I think?